src/components/content/common/deployedServiceDetailsByServiceIdQuery/useServiceDetailsByServiceIdQuery.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
the folder naming structure is bit odd. I did not realize it in the previous PRs.
Could you please change it like this?

common/queries/$queryName.ts -> basically one file for each query. no need of separate folder for each query.

Reply:
good advice, refactored.